README.md

```diff
@@ -1,65 +1,179 @@
-The focus for this challenge is to write high-quality code.
-
-In order to do this, you may pay particular attention to the following:
-* Using diagramming to plan your approach to the challenge
-* TDD your code
+In order to do this, I aimed to give particular attention to :
+* Using diagramming to plan my approach to the challenge
+* TDD the code
 * Focus on testing behaviour rather than state
 * Commit often, with good commit messages
-* Single Responsibility Principle and encapsulation
-* Clear and readable code
-
-## Bowling — how does it work?
-
-### Strikes
-
-The player has a strike if he knocks down all 10 pins with the first roll in a frame. The frame ends immediately (since there are no pins left for a second roll). The bonus for that frame is the number of pins knocked down by the next two rolls. That would be the next frame, unless the player rolls another strike.
-
-### Spares
-
-The player has a spare if the knocks down all 10 pins with the two rolls of a frame. The bonus for that frame is the number of pins knocked down by the next roll (first roll of next frame).
-
-### 10th frame
-
-If the player rolls a strike or spare in the 10th frame they can roll the additional balls for the bonus. But they can never roll more than 3 balls in the 10th frame. The additional rolls only count for the bonus not for the regular frame count.
-
-    10, 10, 10 in the 10th frame gives 30 points (10 points for the regular first strike and 20 points for the bonus).
-    1, 9, 10 in the 10th frame gives 20 points (10 points for the regular spare and 10 points for the bonus).
-
-### Gutter Game
-
-A Gutter Game is when the player never hits a pin (20 zero scores).
-
-### Perfect Game
-
-A Perfect Game is when the player rolls 12 strikes (10 regular strikes and 2 strikes for the bonus in the 10th frame). The Perfect Game scores 300 points.
-
-In the image below you can find some score examples.
-
-More about ten pin bowling here: http://en.wikipedia.org/wiki/Ten-pin_bowling
+* Single Responsibility Principle and encapsulation (WIP -  I plan to extract a frame class that will inject instances of frames into an array in the scorecard class)
+* Clear and readable code (WIP - I plan to refactor further)
+
+## Planning and approach
+
+I created [user stories](./planning/user_stories.md), which in turn helped me establish some models (see [planning folder](./planning)) e.g.: 
+![frame model](./planning/frame_model.svg)
+
+## User stories and commands
+#### Normal frames (where player scores less than 10 in a frame)
+For this user story:
+> So that I don’t get access to more than 10 pins per frame,  
+> I want to know if I’ve tried to enter more than 10 in total across the rolls of a frame (in frames 1-9). 
+
+In irb, enter a pin score greater than 10 for either a roll or a frame:   
+`s.roll_1(11)`,   
+or `s.roll_1(1)` followed by `s.roll_2(10)`.  
+You should get an output like: `RuntimeError (Max pins exceeded, recheck and try again)`  
```

Comment:
Love how you've written your user stories and then planned your functionality immediately below them. Might have to do this myself next time!